src/middlewares.js

```diff
@@ -110,6 +110,11 @@ export function handleParseHeaders(req, res, next) {
   req.config = new Config(info.appId, mount);
   req.info = info;
 
+  const ip = req.ip;
```

Comment:
Through a reverse proxy, yeah the IP would probably be that of the balancer :/. Although in some cases the requesting IP would be passed along in the `X-Forwarded-For` header, not necessarily guaranteed however. To note this _could_ be more than one IP, depending on how many proxies it passed through.

Perhaps we could check for that header and use it's value instead? When present that is.